helpers.py

```diff
@@ -37,6 +37,12 @@ def put(conn, file, key_filename=None, local_path=".", remote_path="."):
     conn.local("scp %s -C -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -P %s %s %s@%s:%s" %
           (key_arg, conn.port, os.path.join(local_path, file), conn.user, conn.host, remote_path))
 
+def run(conn, command, key_filename=None, warn=False):
+    key_arg = _prepare_key_arg(key_filename)
+    result = conn.local("ssh %s -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout=60 -p %s %s@%s %s" %
+                        (key_arg, conn.port, conn.user, conn.host, command), warn=warn)
+    return result
```

Comment:
I sense Fabric is being more and more marginalized in the company. I'm not opposing it though! :-)

Reply:
Hehe, that is right.

The problem I faced is that being a rofs the SSH generated keys at boot will not be saved, so after reboot Fabric will fail in making new SSH connections.